src/guide/jbs-jdk-bug-system.md

```diff
@@ -5,52 +5,290 @@
+* Including the java -fullversion is encouraged for bugs in the JVM, hangs, network issues where the exact version can be critical to determine what fixes may be missing from an older version.
 * Always file separate bugs for different issues.
-  * If two crashes looks related but not similar enough to for sure be the same, it's easier to close a bug as a duplicate than it is to extract the relevant info from a bug to create a new one later.
+    * If two crashes looks related, but not similar enough to be sure they are the same, it's easier to later close a bug as a duplicate than it is to separate out one bug into two.
+
+Things to keep in mind when requesting an improvement:
+
+* Unlike reporting a problem, when it comes to improvements what constitutes a reasonable request can take discussion, and in general it is encouraged that the mailing list for the area is used to suggest an improvement before filing.
+* Enhancements to The Java Language Specification and The JVM Specification are managed through the [Java Community Process](http://jcp.org/)
 
 To find out which component to use for different bugs, consult the [directory to area mapping](#directory-to-area-mapping).
 
+### Implementing a Large Change
+When managing the work for a large change, especially when the work involves multiple engineers, it is recommended that the work is distributed across one or more "implementation" issues which should be linked to the main [Enhancement]{.jbs-field} with a "blocks" link along with any relevant CSRs. The [Enhancement]{.jbs-field} should not be considered done until all the blocking element are completed.  The use of subTasks for [Enhancement]{.jbs}s is not recommended unless all the [Sub-task]{.jbs-field}s are relevant to the fix, if it were to be backported.
+
+### Implementing a JEP
+It recommended that for [JEP]{.jbs-field}s the implementation is spread across one or more [Enhancement]{.jbs-field}s as above.
+
+## Issue states
+
+JBS only has a few states in which an issue can be in:
+
+<table class="dictionary" summary="JBS Label Dictionary">
+  <tr style="text-align:left;"><th>Type</th><th>Covers</th></tr>
+<tr>
+    <td class="dictionary">[New]{.jbs-field}</td>
+    <td class="dictionary">Initial state after an issue is filed. [Bugs]{.jbs-field} in the JDK Project must be taken out of the [New]{.jbs-field} state (Triaged - see below) in a timely manner. In general, this is recommended to be done for all issue types and projects as a sign that the issue is correctly filed, and will be seen by the right group - this is especially important towards the end of a release.</td>
+</tr>
+<tr>
+    <td class="dictionary">Open</td>
+    <td class="dictionary">Most straightforward issues stay in this state until they are closed.  If the issue has some attention then use "In Progress" to show more clearly that the issue is being worked</td>
```

Comment:
Fair point - have updated it to:
Once the issue has been triaged it will be in the [Open]{.jbs-field} state and will stay here until the assignee decides to work on it, at which point it is encouraged that the "Start Work" option be selected to move it to [In Progress]{.jbs-field}